python/ccf/ledger.py

```diff
@@ -162,7 +150,7 @@ def _byte_read_safe(file, num_of_bytes):
     return ret
 
 
-class NodeStatus(IntEnum):
+class NodeStatus(Enum):
     """These are the corresponding status meanings from the ccf.nodes table"""
 
     PENDING = 0
```

Comment:
Shouldn't these now be strings? I assume the integer values were the raw message pack encoding, but these will now be JSON so should be converted from strings?